### PR TITLE
readme, tispark: update TiSpark and enable sparkR and pyspark

### PR DESCRIPTION
**Pull request by tennix**
This PR update TiSpark to 1.0 and also enable easy access to sparkR and pyspark. PTAL @lilin90 @c4pt0r

### CODE REVIEW

**Comment by tennix** [comment]
@lilin90 files in tispark/R are copied from [here](https://github.com/pingcap/tispark/tree/master/R)

**Comment by lilin90** [comment]
@tennix Then that file needs to be improved too.

File: tispark/R/DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Package: TiSparkR
+Type: Package
+Title: TiSpark for R
+Version: 1.1
+Author: PingCAP
+Maintainer: Novemser <novemser@gmail.com>
+Description: A shabby thin layer to support TiSpark in R language.
```
Comment (Hoverbear) [inline tispark/R/DESCRIPTION]: Hmmm... "shabby" generally has a fairly negative connotation. Are you sure that's what is desired?

**Comment by tennix** [comment]
@zhexuany Perhaps we should add a Dockerfile in tispark repo.

**Pull request merged**